### ISSUE TITLE
add  a  c++ program in  .cpp  format

### ISSUE THREAD

**Issue by Shubham56-droid** [Shubham56-droid/DSA#3]
Program to find postorder traversal of BST from preorder traversal ??

**Comment by Lokesh-Rathore** [comment]
Hey bro can I do this task.

**Comment by Lokesh-Rathore** [comment]
Bro I can start this if you assign me.

**Comment by Shubham56-droid** [comment]
yes you can try to put program according to folder

**Comment by Lokesh-Rathore** [comment]
Do I directly makke Pull request.

**Comment by Shubham56-droid** [comment]
yes add program according to folder name like in array folder any solved question related to array

**Comment by Shubham56-droid** [comment]
send the pr accordingly star this repo and follow me i am going to update this repo ur pr will be  merged

**Comment by Lokesh-Rathore** [comment]
In which folder should I put Postorder traversal

**Comment by Lokesh-Rathore** [comment]
And can I add more than one file for c++

**Comment by Shubham56-droid** [comment]
yes u can  add  it directly into main repo

**Comment by Lokesh-Rathore** [comment]
I have sent all the question which you asked.Can you review it.

**Comment by Shubham56-droid** [comment]
yes its merged

**Comment by oneamanish** [comment]
see my pull req.
